opm/common/OpmLog/LogBackend.cpp

```diff
@@ -27,18 +27,33 @@ namespace Opm {
     {
     }
 
+    LogBackend::~LogBackend()
+    {
+    }
 
-    bool LogBackend::includeMessage(int64_t messageFlag) {
-        if (((messageFlag & m_mask) == messageFlag) &&
-            (messageFlag > 0))
-            return true;
-        else
-            return false;
+    void LogBackend::configureDecoration(std::shared_ptr<MessageFormatterInterface> formatter)
+    {
+        formatter_ = formatter;
```

Comment:
But `this->` is not enforced in any way. Also I don't understand
> doesn't encourage redundancy in parameter naming

This has been a very long-standing standard for many codes, not just OPM, so if you want me to adopt a new point of view, point me to the links I should read :-)